runtimevar/runtimevar.go

```diff
@@ -509,9 +509,6 @@ func DecryptDecode(ctx context.Context, k *secrets.Keeper, post Decode) Decode {
 }
 
 // DecoderByName returns a *Decoder based on decoderName.
-// It is intended to be used by VariableURLOpeners in driver packages.
```

Comment:
Let's keep this line in the docstring, to let end users know they shouldn't be using this directly.

Reply:
Done